README.md

```diff
@@ -1,2 +1,111 @@
+
+
+## Commands
+
+| Special Permissions | Category | Name (prefixes) | Description | Usage |
+| --------- | --------- | --------- | --------- | --------- | 
+| Owner Only | Developer | **Sync** | Sync all the slash commands globally | `{prefix}sync` |
+| Owner Only | Developer | **load** | Load a Cog. | `{prefix}load [cog]` |
+| Owner Only | Developer | **unload** | Unload a Cog. | `{prefix}unload [cog]` |
+| Owner Only | Developer | **reload** | Reload a Cog. | `{prefix}reload [cog]` |
+| Owner Only | Developer | **loadall** | Load all Cog. | `{prefix}loadall` |
+| Owner Only | Developer | **unloadall** | Unload all Cog. | `{prefix}unloadall` |
+| Owner Only | Developer | **reloadall** | Reload all Cog. | `{prefix}reloadall` |
+| Owner Only | Developer | **getusermetric** | Get User Metrics. | `{prefix}getusermetric [member]` |
+| None | Fun | **trash** | Throw someone in the trash. | `{prefix}trash [user]` |
+| None | Fun | **number** | Gets a random number. | `{prefix}number`: *will get a random number* <br> `{prefix}number [number]`: *will get the [number]* |
+| None | Fun | **meme** | Gets a random meme. | `{prefix}meme`: *will get a random meme* |
+| None | Fun | **joke** | Gets a random joke. | `{prefix}meme`: *will get a random joke* |
+| None | Fun | **eightball** | Returns a random response. | `{prefix}eightball [question]`: *will return a random response* |
+| None | Fun | **token** | Generates a random token. | `{prefix}token`: *will generate a token* |
+| None | Fun | **binary** | Commands for binary | `{prefix}binary` |
+| None | Fun | **binary encode** | Encodes plaintext and return binary | `{prefix}binary [text]`: *return encoded text* |
+| None | Fun | **binary decode** | Decode binary text to plaintext | `{prefix}binary [text]`: *return plaintext* |
+| None | Fun | **reverse** | Reverse inputted string | `{prefix}reverse [string]`: *returns string reversed* | 
+| None | Fun | **owofy** | Owofy inputted string | `{prefix}owofy [text]`: *returns owofy text* |
+| None | Fun | **mock** | Mockify inputted string | `{prefix}mock [text]`: *returns mocked text* |
+| None | Fun | **beerparty** | Start a beerparty 🍻!! | `{prefix}beerparty [reason]`: *start a beer party* |
+| None | General | **source** (github, code) | Get the source of this bot | `{prefix}source` *will send link to my source code* <br > `{prefix}source [command]` *will send link to the source code of the command* <br> `{prefix}source [command] [subcommand]`: *will send link to the source code of the subcommand* | 
+| None | General | **define** | Gets deinitions from Urban Dictionary. | `{prefix}define [word]`: *will send the definition of the word* |
+| None | General | **avatar** | Commands for getting avatars. | `{prefix}avatar`: *will send a list of available methods* |
+| None | General | **avatar main** | Start a beerparty 🍻!! | `{prefix}avatar main [user]`: *will send the main avatar of the user* |
```

Comment:
fix the bug here:
![image](https://github.com/The-Coding-Realm/coding-bot-v6/assets/77532758/a2798169-2bf9-4f22-bd44-6f2aa4de2a58)